zap/src/main/java/org/zaproxy/zap/extension/script/ScriptUI.java

```diff
@@ -131,6 +131,8 @@ default void removeSelectionListener(TreeSelectionListener tsl) {
 
     void displayScript(ScriptWrapper script);
 
+    void displayScript(ScriptWrapper script, boolean enableFocus);
```

Comment:
Add default implementation which calls existing `displayScript`.

Reply:
I was thinking opposite. Shouldn't it be like the existing display script should have a default implementation which will call the newly added display script? Because we need to prune this setfocus boolean to setscript also.

Reply:
Shouldn't it be the opposite? I was thinking to add a default implementation for `void displayScript(ScriptWrapper script);`. Am I thinking wrong?

Reply:
The existing implementations don't implement the new method yet, any code calling it will lead to `AbstractMethodError`. The default implementation addresses that (and calling the existing method keeps the current behaviour). Also, worth renaming the method to `displayScriptWithouFocus` and remove the `boolean`.

Reply:
okay got it. I have updated the code please check

Reply:
Sorry, lets go back to `void displayScript(ScriptWrapper, boolean)` to keep it closer to what we do in the rest of the codebase.

Reply:
sorry didn't see this

Reply:
No worries :)